clients/banking/src/components/TransactionDetail.tsx

```diff
@@ -569,62 +569,60 @@ export const TransactionDetail = ({ transaction, large }: Props) => {
                   {rejectedReason}
                   {transactionId}
 
-                  {originTransaction != null && (
-                    <>
-                      <LakeLabel
-                        type="viewSmall"
-                        label={t("transaction.originalTransactionId")}
-                        render={() => (
-                          <LakeText variant="regular" color={colors.gray[900]}>
-                            {originTransaction.id}
-                          </LakeText>
-                        )}
-                      />
+                  {originTransaction != null && [
```

Comment:
doesn't this warn without items having `key`?

Reply:
it doesn't on my side (I hope my linter is well configured ?)
I'm not iterating over the array, I'm rendering it as `ReadOnlyFieldList` child. And `ReadOnlyFieldList` wraps its Childs in Views with keys :

<img width="597" alt="image" src="https://github.com/swan-io/swan-partner-frontend/assets/26482371/60b8b7d4-3500-431a-ab87-f9b426bae43c">
<img width="1180" alt="image" src="https://github.com/swan-io/swan-partner-frontend/assets/26482371/3c393511-4ac2-4769-bae1-d7e2529528eb">

do you think I should add a key anyway ?

Reply:
oh right, all good then!